index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="index.css">
+        <script src="https://kit.fontawesome.com/5031df2cf9.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <header>
+          <nav>
+            <div class="logo">
+                My Logo
+            </div>
+            <div class="menu-bar">
+                <i class="fa-solid fa-bars"></i>
+            </div>
+            <div class="drop-down">
+                <ul>
+                    <li>first item</li>
+                    <li>second item</li>
+                    <li>third item</li>
+                </ul>
+            </div>
+          </nav>
+        </header>
+       
+       
+          <section class="headline-section"> 
+            <div>
+                <h1>I’m Trust Glad to see you!</h1>
+            </div>
+            <div>
+                <p>
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                    experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="lets-connect">
+                    <h4>LETS CONNECT</h4>
+                    <div class="social-icons">
+                        <a href="#">
+                          <i class="fa-brands fa-twitter"></i>
+                        </a>
+                        <a href="#">
+                          <i class="fa-brands fa-linkedin-in"></i>
+                        </a>
+                        <a href="#">
+                          <i class="fa-solid fa-m"></i>
+                        </a>
+                        <a href="#">
+                          <i class="fa-brands fa-github"></i>
+                        </a>
+                        <div>
+                          <i class="fa-regular fa-hand-peace"></i>
+                        </div>
+                    </div>
```

Comment:
- [ ] The list of icons should have a ul tag to group them, and li per icon; each icon has a link (a tag). That would be semantically correct.